data/block/block.go

```diff
@@ -79,6 +75,11 @@ func (h *Header) SetChainID(chainID []byte) {
 	h.ChainID = chainID
 }
 
+// SetSoftwareVersion sets the chain ID on which this block is valid on
+func (h *Header) SetSoftwareVersion(version []byte) {
```

Comment:
there is no need for this Set function.

Reply:
Actually it is used in an integration test. Can be useful also elsewhere